### ISSUE TITLE
Typescript definition missing ready() method

### ISSUE THREAD

**Issue by thaoula** [bee-queue/bee-queue#132]
Hi Guys,

It seems that the typescript definition is missing the ready callback -

this code is found in queue.js but you cannot call this when using typescript and need to cast to any.

```
  ready(cb) {
    if (cb) this._ready.then(() => cb(null), cb);

    return this._ready;
  }
```

**Comment by skeggse** [comment]
Fixed in #130.

**Comment by hughsw** [comment]
See PR #187
